src/components/AddTodo.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { FormEvent, useState } from 'react'
+import styled from 'styled-components'
+
+interface Props {
```

Comment:
in this scenario, you can use `Type`
interfaces are used when you want to extend a Type (mostly)